sktime/classification/interval_based/tests/test_tsf.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+import pytest
+from sklearn.model_selection import train_test_split
+from sktime.classification.interval_based import TimeSeriesForest
+from sktime.datasets import load_gunpoint
+
+
+gunpoint_probas = np.array(
+    [
+        [1.0, 0.0],
+        [1.0, 0.0],
+        [0.9, 0.1],
+        [1.0, 0.0],
+        [0.0, 1.0],
+        [0.95, 0.05],
+        [0.0, 1.0],
+        [0.9, 0.1],
+        [1.0, 0.0],
+        [0.15, 0.85],
+        [0.8, 0.2],
+        [1.0, 0.0],
+        [1.0, 0.0],
+        [0.25, 0.75],
+        [1.0, 0.0],
+        [0.95, 0.05],
+        [1.0, 0.0],
+        [1.0, 0.0],
+        [0.0, 1.0],
+        [0.0, 1.0],
+    ]
+)
+
+
+@pytest.mark.parametrize("n_jobs", [1, 4])
+def test_on_gunpoint(n_jobs):
+    np.random.seed(42)
```

Comment:
We don't need to call `np.random.seed` here, but need to pass it to `train_test_split`